schema-extended/Common.json

```diff
@@ -0,0 +1,308 @@
+      "title": "arXiv posting status",
+      "description": "The status of this document version on arXiv.",
+      "type": "string",
+      "enum": [
+        "posted",
+        "withdrawn"
+      ],
+      "minLength": 1
+    },
+    "author_id": {
+      "description": "arXiv author identifier, if available.",
+      "type": "string",
+      "pattern": "^[a-z]+_[a-z]{1}_[0-9]+$"
+    },
+    "comment": {
+      "title": "Comment",
+      "type": "string",
+      "minLength": 1
+    },
+    "conference_info": {
+      "title": "Conference information",
+      "type": "object",
+      "properties": {
+        "conference_titles": {
+          "title": "List of conference titles",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "minLength": 1
+          },
+          "minItems": 1,
+          "uniqueItems": true
+        },
+        "conference_pid": {
+          "title": "Conference Persistent Identifier",
+          "$comment": "placeholder to support: https://www.crossref.org/working-groups/conferences-projects/",
+          "description": "The persistent identifier for the conference.",
+          "type": "string",
+          "minLength": 1
+        }
+      },
+      "required": [
+        "conference_titles"
+      ]
+    },
+    "collaboration": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The fullest representation of a collaborations's name",
+          "allOf": [
+            {
+              "type": "string",
+              "minLength": 1
+            },
+            {
+              "not": {
+                "type": "string",
+                "pattern": "(?i)(collaboration)"
+              }
+            }
+          ]
+        }
+      },
+      "required": [
+        "name"
+      ],
+      "additionalProperties": false
+    },
+    "document_type": {
+      "title": "Document type",
+      "$comment": "TODO: enumeration needs further review, and will need to decide whether the items require their own definitions with more structure.",
+      "type": "string",
+      "enum": [
+        "article",
+        "autobiography",
+        "biography",
+        "book",
+        "book chapter",
+        "book review",
+        "conference proceeding",
+        "conference proceeding index",
+        "erratum",
+        "memorial",
+        "thesis",
+        "lecture notes",
+        "whitepaper"
+      ]
+    },
+    "doi": {
+      "title": "DOI",
+      "description": "Digital Object Identifier.",
+      "type": "string",
+      "pattern": "^10\\.\\d+(\\.\\d+)?/\\S+$",
+      "minLength": 1
+    },
+    "external_publication_status": {
+      "title": "External publication status",
+      "description": "The publication status of this document version.",
+      "type": "string",
+      "enum": [
+        "preprint",
+        "submitted",
+        "accepted",
+        "published"
+      ],
+      "$comment": "Is this enough?"
+    },
```

Comment:
This definition is included as a property the `Version.json` schema.

I don't recall whether the group had provided any specific feedback on this in previous calls.